platforms/quorum/configuration/roles/helm_component/templates/constellation.tpl

```diff
@@ -54,7 +54,11 @@ spec:
 {% endfor %}
 {% endif %}
     constellation:
+{% if network.config.tm_tls == 'strict' %}
+      url: https://{{ peer.name }}.{{ external_url }}:{{ peer.transaction_manager.ambassador }}/
```

Comment:
does this work?

Reply:
yes, in netwok.yaml we are specifying http for tm_nodes that goes to othernodes placeholder and I think that's why it is working.